servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -89,6 +91,10 @@ public TieredBlockStore(TachyonConf tachyonConf) throws IOException {
     mMetaManager = BlockMetadataManager.newBlockMetadataManager(mTachyonConf);
     mLockManager = new BlockLockManager(mMetaManager);
 
+    // initially use empty lists to provide full view
+    BlockMetadataManagerView initView = new BlockMetadataManagerView(mMetaManager,
+        new HashSet<Integer>(), new HashSet<Long>());
```

Comment:
Yes, but I'll remove them for now and add these lines when I update the evictors and allocators.